### PR TITLE
Turns off lazyInit for classes with @Scheduled methods. 

### PR DESCRIPTION
**Pull request by slackboxster**
Improvement for #2216.

### CODE REVIEW

File: grails-spring/src/main/groovy/org/grails/spring/DefaultBeanConfiguration.java [hunk 1]
```diff
@@ -230,6 +237,19 @@ protected AbstractBeanDefinition createBeanDefinition() {
         return bd;
     }
 
+    private boolean hasScheduledMethod(Class clazz) {
```
Comment (slackboxster) [inline grails-spring/src/main/groovy/org/grails/spring/DefaultBeanConfiguration.java]: I created these methods to clean things up, but feel like they don't fit in this class. Should these go somewhere else?

Comment (jeffbrown) [inline grails-spring/src/main/groovy/org/grails/spring/DefaultBeanConfiguration.java]: This could probably be done with a bean definition post processor.

**Comment by bobbywarner** [comment]
As mentioned above, this can be accomplished in alternate ways.  So, closing this PR as it's not needed.

**Comment by slackboxster** [comment]
Sounds good.

**Pull request closed without merging**